app/services/proposal_update_recorder.rb

```diff
@@ -24,7 +24,11 @@ def changed_attributes
   end
 
   def update_comment_format(key)
-    "#{bullet}*#{property_name(key)}* was changed " + former_value(key) + "to #{new_value(key)}"
+    if key !~ /id/
```

Comment:
maybe switch the order in order to make this a positive assertion instead of a negative one? i.e. `if key =~ /id/` first.